distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.web.cfg

```diff
@@ -38,3 +38,15 @@ org.ops4j.pax.web.ssl.keypassword=${javax.net.ssl.keyStorePassword}
 
 # Client authenticate at server is wanted
 org.ops4j.pax.web.ssl.clientauthwanted=true
+
+#Included SSL/TLS Cipher Suites Regular Expressions
+org.ops4j.pax.web.ssl.ciphersuites.included=${https.cipherSuites}
+
+#Excluded SSL/TLS Cipher Suites Regular Expressions
+org.ops4j.pax.web.ssl.ciphersuites.excluded=.*RC4.*,SSL.*,.*anon.*,.*3DES.*
```

Comment:
This configuration file can be edited as needed for compatibility.  Exclusions are partly based on NIST recommendations.